modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-test/src/testFunctional/paths/ProcessBuilderKaleoDesignerReact.path

```diff
@@ -64,7 +64,16 @@
 	<td>//div[@class='dropdown-menu show']</td>
 	<td></td>
 </tr>
-
+<tr>
+	<td>LANGUAGE_TRANSLATION_IN_DROPDOWN</td>
+	<td>//div[@class='dropdown-menu show']//*[contains(@class,'lexicon-icon-${key_languageTranslationOption}')]</td>
```

Comment:
This path it's not needed. Here you could use the path that it's already created : `TRANSLATION_COUNTRY_LABEL`.